backend/modelsClass/index.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
```

Comment:
noticed this `.js` file which seems to be the output of `tsc` "compiling" a `.ts` file 🤔 if you're going to use TypeScript, these files should not be part of your repository, since running the `tsc` command will produce them (for build purposes); you're free to use either TypeScript/JavaScript, but if you do choose to continue with TypeScript, make sure it won't be additional load for your entire team 🫡